src/main/java/com/kodilla/ecommercee/domain/Product.java

```diff
@@ -1,9 +1,40 @@
 package com.kodilla.ecommercee.domain;
 
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.math.BigDecimal;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity(name = "PRODUCTS")
 public class Product {
+
+
+    @Id
+    @NotNull
+    @GeneratedValue
+    @Column(name = "PRODUCT_ID", unique = true)
     private Long id;
+
+    @Column(name = "NAME")
     private String name;
+
+    @Column(name = "DESCRIPTION")
     private String description;
-    private Double price;
-    private Long groupId;
+
+    @Column(name = "PRICE")
+    private BigDecimal price;
+
+    @ManyToOne
+    @Column(name = "GROUP_ID")
+    private GroupId groupId;
```

Comment:
Jeśli chcesz utworzyć relację z Group to nazwa powinna być właśnie Group. Stosując podejście podobne do innych pull requestów, powininna być używa właściwa nazwa (może to być zakomenowane jeśli nie istnieje w głównym branchu)